receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,533 @@
+    type: string
+  vcenter.resource_pool.name:
+    description: The name of the resource pool
+    type: string
+  vcenter.datastore.name:
+    description: The name of the vCenter datastore
+    type: string
+  vcenter.vm.name:
+    description: The name of the virtual machine
+    type: string
+  vcenter.vm.id:
+    description: The instance UUID of the virtual machine
+    type: string
+
+attributes:
+  disk_state:
+    description: The state of storage and whether it is already allocated or free.
+    type: string
+    enum:
+      - available
+      - used
+  host_effective:
+    value: effective
+    description: Whether the host is effective in the vCenter cluster
+    type: bool
+    enum:
+      - "true"
+      - "false"
+  latency_direction:
+    value: direction
+    description: The direction of disk latency.
+    enum:
+      - read
+      - write
+  latency_type:
+    value: type
+    description: The type of disk latency being reported.
+    enum:
+      - kernel
+      - device
+  throughput_direction:
+    value: direction
+    description: The direction of network throughput.
+    type: string
+    enum:
+      - transmitted
+      - received
+  vm_count_power_state:
+    value: power_state
+    description: Whether the virtual machines are powered on or off
+    type: string
+    enum:
+      - "on"
+      - "off"
+  vsan_latency_type:
+    value: type
+    description: The type of vSAN latency.
+    type: string
+    enum:
+      - read
+      - write
+  vsan_operation_type:
+    value: type
+    description: The type of vSAN operation.
+    type: string
+    enum:
+      - read
+      - write
+      - unmap
+  vsan_throughput_direction:
+    value: direction
+    description: The type of vSAN throughput.
+    type: string
+    enum:
+      - read
+      - write
+
+metrics:
+  vcenter.cluster.cpu.available:
+    enabled: true
+    description: The amount of CPU available to the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.effective:
+    enabled: true
+    description: The effective CPU available to the cluster. This value excludes memory from hosts in maintenance mode or are unresponsive.
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.cpu.used:
+    enabled: true
+    description: The amount of CPU used by the cluster
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.available:
+    enabled: true
+    description: The available memory of the cluster.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.memory.effective:
+    enabled: true
+    description: The available memory of the cluster.
```

Comment:
I can't tell the difference between these two.

Reply:
Ahh description mishap effective memory is the available to use memory (i.e memory of all active running hosts to provision). This thread sort of cleared up my confusion on it

https://communities.vmware.com/t5/VMware-PowerCLI-Discussions/What-is-effective-memory-effective-CPU-in-Get-Cluster-name-xxx/td-p/2724919